inference-engine/src/transformations/include/transformations/convert_opset1_to_legacy/convert_mul_add_to_scaleshift_or_power.hpp

```diff
@@ -36,4 +36,4 @@ enum class CONVERSION_RESULT {
 };
 
 INFERENCE_ENGINE_API_CPP(CONVERSION_RESULT)
-check_constant(const std::shared_ptr<ngraph::op::Constant> & constant, const ngraph::Shape & shape);
+check_constant(const std::shared_ptr<ngraph::op::Constant> & constant, const ngraph::PartialShape & shape);
```

Comment:
Consider fixing CVS-32076 as part of this MR as well.

Reply:
I looked into this problem and it requires significant changes. I'll do it in separate PR.

Reply:
Finally decided to have it in this PR. Fix is ready.

Reply:
Great!